src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -76,6 +75,7 @@ internal static TypeSymbol CopyTypeCustomModifiers(TypeSymbol sourceType, TypeSy
             // NOTE: ref is irrelevant here since we are just encoding/decoding the type out of the signature context
             ImmutableArray<bool> flags = CSharpCompilation.DynamicTransformsEncoder.EncodeWithoutCustomModifierFlags(destinationType, refKind);
             TypeSymbol typeWithDynamic = DynamicTypeDecoder.TransformTypeWithoutCustomModifierFlags(sourceType, containingAssembly, refKind, flags);
+            // PROTOTYPE: Should we use NativeIntegerTypeDecoder here?
```

Comment:
>PROTOTYPE [](start = 15, length = 9)

We usually add a feature name in parens.

Reply:
The feature name shouldn't be necessary since `PROTOTYPE` comments exist only in feature branches.

---
In reply to: [372625000](https://github.com/dotnet/roslyn/pull/41160#discussion_r372625000) [](ancestors = 372625000)

Reply:
There do exist a few in master.

---
In reply to: [372656569](https://github.com/dotnet/roslyn/pull/41160#discussion_r372656569) [](ancestors = 372656569,372625000)